index.html

```diff
@@ -95,84 +95,7 @@ <h2 class="mobile-work-header">My Recent </br> Works</h2>
         <h2 class="desktop-work-header">My Recent Works</h2>
     </div>
     <div class="myworks-wrapper">
-        <div class="myworks-card">
-        <div class="myworks-thumbnail">
-            <img  alt="">
-            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
-            <ul class="work-languages" >
-                <li>Ruby on Rails</li>
-                <li>CSS</li>
-                <li>JavaScript</li>
-                <li>HTML</li>
-            </ul>
-            <button type="button" > See Project</button>
```

Comment:
-  [ ] Good work on your pop up but there are some aspects you can improve on :
  - Your mobile version appears to be different. Kindly consider adjusting some of these aspects
    - Height of the image. Yours appears to be shorter
    
       

    - The elements containing the languages appear to be different. Kindly center the texts

        

    - The `see live` and `see more ` buttons background color appear to be different from the original design
    
![image](https://github.com/demesameneshoa/Portfolio/assets/98453979/304caae3-67c3-46f4-a88e-571b85f2d7c5)

### Checkout the original design

![image](https://github.com/demesameneshoa/Portfolio/assets/98453979/10c90177-ef66-43ce-a607-e6c28efb80ad)